src/Workspaces/Core/Portable/Differencing/LongestCommonSubsequence.cs

```diff
@@ -179,6 +179,15 @@ protected double ComputeDistance(TSequence oldSequence, int oldLength, TSequence
                 return (oldLength == newLength) ? 0.0 : 1.0;
             }
 
+            // If the sequences differ significantly in size their distance will be very close to 1.0 
+            // (even if one is a strict subsequence of the other).
+            // Avoid running an expensive LCS algorithm on such sequences.
+            double lenghtRatio = (oldLength > newLength) ? oldLength / newLength : newLength / oldLength;
```

Comment:
>lenght [](start = 19, length = 6)

typo

Reply:
I'll fix that in a follow-up change to avoid restarting tests.